.gitignore

```diff
@@ -11,10 +11,4 @@
 /.python-version
 
 # documentation
-/doc/_build/
-/doc/*.yml.rst
-/doc/role_overview.rst
-/doc/sphinx-template/
-
-# vagrant
-/.vagrant/
+/doc/
```

Comment:
Why do you ignore the whole `/doc/`? There are some required files (e.g. `index.rst`). The old configuration just ignored the generated documents.

Reply:
Fixed and squashed, git horray!